From cc062ca36e070139cc1ecbc7e2f7576381c6ad3c Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Mon, 22 Feb 2021 11:11:40 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3568: update reg id for rgb and lvds

the reg id will be used to find port id at uboot logo

Change-Id: If32ec28a3b1d30d20ded4229a767777bba0ea07a
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3568.dtsi | 13 ++++++-------
 1 file changed, 6 insertions(+), 7 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568.dtsi b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
index f05cba67201b..bc8aeeecebff 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
@@ -673,13 +673,13 @@
 					#address-cells = <1>;
 					#size-cells = <0>;
 
-					lvds_in_vp1: endpoint@0 {
-						reg = <0>;
+					lvds_in_vp1: endpoint@1 {
+						reg = <1>;
 						remote-endpoint = <&vp1_out_lvds>;
 					};
 
-					lvds_in_vp2: endpoint@1 {
-						reg = <1>;
+					lvds_in_vp2: endpoint@2 {
+						reg = <2>;
 						remote-endpoint = <&vp2_out_lvds>;
 					};
 				};
@@ -701,12 +701,11 @@
 					#address-cells = <1>;
 					#size-cells = <0>;
 
-					rgb_in_vp2: endpoint@0 {
-						reg = <0>;
+					rgb_in_vp2: endpoint@2 {
+						reg = <2>;
 						remote-endpoint = <&vp2_out_rgb>;
 					};
 				};
-
 			};
 		};
 
-- 
2.35.3

